lib/shopify_api/resources/vault_session.rb

```diff
@@ -0,0 +1,28 @@
+module ShopifyAPI
+  module VaultSessionFormat
```

Comment:
Because updating cardsink to include a `root` based on some criteria and/or update all the existing apps to use the new format is an enormous task.

I can extract this into a separate file, but since this was only used by this single class, I've never see any benefits to do so.